.gitignore

```diff
@@ -4,10 +4,8 @@ compile_commands.json
 codeql-db/
 build/
 docs/doxygen/
-tests/plt*
```

Comment:
Now there is an extra change here. I think you meant to revert the earlier change, but that's not what happened.

Reply:
Did that fix it?

Reply:
No... When you click on 'Files changed' on the PR, you should not see the .gitignore file there.

Reply:
The easiest way to fix it is to checkout this file from before you changed it: https://stackoverflow.com/questions/215718/how-do-i-reset-or-revert-a-file-to-a-specific-revision
and then make a new commit.